guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -18,7 +18,7 @@ include::snip_firewalld.adoc[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install} dhcp bind
+# {package-install} dhcp bind-utils
```

Comment:
I could reproduce `bind-utils`, but not `dhcp`. There is both `dhcp-client` and `dhcp-server` on EL8

Reply:
Good catch. I checked, and found out the same. `dhcp-server` is the one that we need since we are setting up a DHCP server as per the procedure. However, I could now notice something that we missed earlier. The procedure is for installing an external DHCP server to use with Satellite or Capsule server. It seems that majority of readers of the procedure (including me) got the impression that the packages are to be installed on the Satellite or Capsule server itself, when they are actually supposed to be installed on a separate RHEL system.

So, depending on the release of the RHEL system to be set up and configured as the external DHCP server, the packages that need to be installed will differ:
 - On a RHEL 7.x system, 'dhcp' and 'bind' packages are to be installed.
 - On RHEL 8.x and RHEL 9.x, the 'dhcp-server' and 'bind-utils' packages are to be installed.

Now, the question is: which of these shall we include in the procedure? RHEL 8.x and 9.x only, or RHEL 7.x as well?

Also, I would suggest to explicitly refer to the RHEL host on which the packages to be installed in step (1) as 'On your Red Hat Enterprise Linux host that is going to be configured as the external DHCP server' rather than just 'On your Red Hat Enterprise Linux host', to clear out any confusion.

Reply:
So, I will be explicitly adding EL7 and EL8 both in downstream 6.11 (another PR). For this PR, I am only considering EL8 and hence, only working at package level.

Reply:
Addressed :+1: